spec/spec_helper.rb

```diff
@@ -3,9 +3,10 @@
   SimpleCov.start
 end
 
-VCR.configure do |config|
-  config.ignore_hosts 'codeclimate.com' if ENV['CI']
-  config.cassette_library_dir = File.join(ManageIQ::Providers::Azure::Engine.root, 'spec/vcr_cassettes')
-end
+# Uncomment in case you use vcr cassettes
```

Comment:
We want to run vcr cassettes here.
Thats another thing that could be checked dynamically